upgrading/automated_upgrades.adoc

```diff
@@ -154,63 +228,93 @@ To prepare for an automated upgrade:
+. For any upgrade path, always ensure that you have the latest version of the
+*openshift-ansible* package on each RHEL 7 system:
++
+----
+# yum update openshift-ansible
+----
++
+[NOTE]
+====
+In previous {product-title} releases, the *atomic-openshift-utils* package was
+installed for this step. However, starting with {product-title} 3.10, that
+package is removed and the *openshift-ansible* package provides all
+requirements.
+====
+
+. If you are upgrading from {product-title} 3.9 to 3.10:
 
-.. Manually disable the 3.7 repository and enable _both_ the 3.8 and 3.9
-repositories on each master and node host. You must also enable the
-*rhel-7-server-ansible-2.4-rpms* repository, which is a new requirement starting
-with {product-title} 3.9:
+.. Manually disable the 3.9 repository and enable the 3.10 repository on each
+master and node host. You must also enable the *rhel-7-server-ansible-2.4-rpms*
+repository, if it is not already:
 +
 ----
-# subscription-manager repos --disable="rhel-7-server-ose-3.7-rpms" \
-    --enable="rhel-7-server-ose-3.8-rpms" \
-    --enable="rhel-7-server-ose-3.9-rpms" \
+# subscription-manager repos --disable="rhel-7-server-ose-3.9-rpms" \
+    --enable="rhel-7-server-ose-3.10-rpms" \
     --enable="rhel-7-server-rpms" \
     --enable="rhel-7-server-extras-rpms" \
     --enable="rhel-7-server-ansible-2.4-rpms" \
     --enable="rhel-7-fast-datapath-rpms"
 # yum clean all
 ----
 
-.. In previous versions of {product-title}, master hosts were marked unschedulable
-by default by the installer, meaning that no pods could be placed on the hosts.
-Starting with {product-title} 3.9, however, masters must be marked schedulable;
-this is done automatically during upgrade.
+.. The upgrade process will be blocked until you have the newly required ConfigMaps
+for bootstrapping nodes in the *openshift-node* project. Run the following
+playbook to have the defaults created for you automatically:
 +
-As a result of this requirement, the default node selector must also now be set
-by default; this is also done automatically during upgrade if it is unset,
-otherwise it is left as is. For either scenario, masters are automatically
-labeled with the `master` node role, and non-master, non-infrastructure nodes
-are labeled with the `compute` node role.
+[subs=attributes+]
+----
+# ansible-playbook -i </path/to/inventory/file> \
+    {pb-prefix}playbooks/openshift-master/openshift_node_group.yml
+----
 +
-See the {product-title} 3.9 Release Notes on the these notable technical changes
-and ensure that you understand their impact on your cluster:
+To verify they were created sucessfully:
 +
-- xref:../release_notes/ocp_3_10_release_notes.adoc#ocp-310-schedulable-masters[Masters Marked as Schedulable Nodes by Default]
-- xref:../release_notes/ocp_3_10_release_notes.adoc#ocp-310-default-node-selector[Default Node Selector Set By Default and Automatic Node Labeling]
-endif::[]
+----
+$ oc get cm -n openshift-node
+----
```

Comment:
Lets add a task to have them actually verify the contents match the expectations, something like
"Carefully verify that each configmap is defined as expected. ex:
`$ oc describe configmaps -n openshift-node node-config-master`
"